CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,489 @@
+---
+CIP: ?
+Title: On-chain dApp Certification Metadata
```

Comment:
I don't think "dApp" should be in the title. For example, is an NFT art project that uses just a minting contract considered a dApp too? While dApps are the main users of this CIP, on-chain certification metadata can also be used for other entities. 
Additionally, why was the term "Standard" redundant when it was used in CIP 25, 27, 68, and 71?

Reply:
I added it because `dApp` appeared 100% applicable from the proposal as written, but yours is a sensible counterexample & grounds for removing it (I will do so now which also respects the fact that the authors didn't draft it that way).  We really do try to make these titles as specific as possible which is becoming more important as the number of CIPs increases.

Likewise in the early days the word "Standard" was more liberally used, before the discussions that came along in the same year the Governance issue became popular and people began paying far more attention to what's implied by calling something a standard.  I wouldn't like to have it here because it suggests there's only one canonical means of representing on-chain certification data.  For others like the NFT proposals (25, 27) the context makes it more clear that these represent choices of what standard to follow.  Conversely if we keep the word "Standard" in this CIP title then we might as well put it in _every_ CIP title.

Reply:
Since our main focus was and is on DApps, the language and tone of this CIP is DApp-based. However, I believe we can retain the structure while changing some words to include any project. If we do that here, we may consider doing it for CIP 72 as well. The terminology we should use is an open discussion.

Regarding removing the word "Standard," I now understand and completely agree with you. Thank you for the explanation.

Reply:
thanks @MIxAxIM - feel free to change the CIP title again without interference when the working group has the particular terminology worked out 😎

Reply:
@MIxAxIM with the PR and CIP titles consistent over a month of editing I assume this issue has been resolved. 😎